docker/benchmark-runner/Dockerfile

```diff
@@ -54,13 +54,15 @@ ENV WORKDIR /out
 RUN mkdir -p $WORKDIR
 WORKDIR $WORKDIR
 
+ARG fuzzer
```

Comment:
This line is unnecessary it already is at the top of the file.

Reply:
It gets reset after that. I couldn't get it to work without this. If I remove it, it basically returns an empty string (at least on a local run).

Reply:
Are you sure? I find this behavior extremely bizarre. Especially given that we use the same variable twice on 37 and 41. If you're sure please just leave a comment explaining why you are doing this.

Reply:
Yes I'm pretty sure this happens (at least on a local run). I'll add the comment.

Reply:
I'll try a CI run without it just in case.

Reply:
Can be seen in [this link](https://github.com/google/fuzzbench/runs/959551133?check_suite_focus=true).
Made changes.